coalib/bears/requirements/PythonRequirement.py

```diff
@@ -25,6 +25,41 @@ def multiple(cls, *args):
         Creates a tuple of multiple ``pip`` requirements, consisting of
         either tuples or strings.
 
+        You should use the ``multiple`` method if you have more
+        requirements from the same manager. This can receive both tuples of
+        strings, in case you want a specific version, or a simple string, in
+        case you want the latest version to be installed.
+
+        This is the case where you would provide strings only, to install the
+        latest version automatically:
+
+        >>> REQUIREMENTS = PythonRequirement.multiple(
+        ...                                     'coala_decorators','setuptools')
+
+        And if you choose to mix them, specifying version for some and for some
+        not:
+
+        >>> REQUIREMENTS = PythonRequirement.multiple(
+        ...                         'coala_decorators',('setuptools', '19.2'))
```

Comment:
and also here some kind of comparing would be good to see what `multiple` actually does